docs/modules/architecture/partials/nav.adoc

```diff
@@ -6,7 +6,7 @@
 ** xref:architecture:event-time-processing.adoc[]
 ** xref:architecture:sliding-window.adoc[]
 ** xref:architecture:in-memory-storage.adoc[]
-
+** xref:architecture:compact-binary-specification.adoc[]
```

Comment:
IMO, this is not really related to architecture

Reply:
Isn't it a design document regarding serialization? So I thought it is related to serialization architecture. Where do you think we should put the document instead?

Reply:
I don't think serialization is part of the architecture. I believe we should have a separate section in the reference tab called sterilization, at put this under it

Reply:
As I said in slack as well, I think we can put it there as well. I will change the location of the document. You can keep the comment unresolved till I do it as a reminder to me.

Reply:
I was simply thinking to put it as a section under Develop Solutions -> Serialization -> Compact Serialization.

If not, there should be a cross-reference to this spec API from the introduction part of the Develop Solutions -> Serialization -> Compact Serialization section.

Reply:
@Serdaro Isn't the document too long for being a section in a page ? I think it should be a separate page. I think cross reference makes more sense for this reason. Where should be the page's location in this case? I think a page directly under "Reference" makes sense. (I am also ok about putting it under architecture obviously)